exporter/elasticsearchexporter/request.go

```diff
@@ -0,0 +1,39 @@
+package elasticsearchexporter
+
+import (
+	"bytes"
+	"context"
+)
+
+type request struct {
```

Comment:
Should we inline the `Export` method into `pushLogsData` and `pushTraceData`? I'm concerned that folks may think that this is a fully fledged exporterhelper request type.

Reply:
Valid concern, I've cleaned that up.